ogame.go

```diff
@@ -4218,6 +4222,18 @@ func (b *OGame) sendFleet(celestialID CelestialID, ships []Quantifiable, speed S
 	}
 
 	fleet1Doc, _ := goquery.NewDocumentFromReader(bytes.NewReader(pageHTML))
+
+	techs := b.extractor.ExtractFleet1ResearchesFromDoc(fleet1Doc)
+	fuelCapacity := ShipsInfos{}.FromQuantifiables(ships).Cargo(Researches{}, true, false, false)
+
+	_, fuel := CalcFlightTime(
+		b.GetCachedCelestialByID(celestialID).GetCoordinate(), where,
+		b.serverData.Galaxies, b.serverData.Systems, b.serverData.DonutGalaxy, b.serverData.DonutSystem, b.serverData.GlobalDeuteriumSaveFactor,
+		float64(speed)/10, GetFleetSpeedForMission(b.IsV81(), b.serverData, mission), ShipsInfos{}.FromQuantifiables(ships), techs, b.characterClass, holdingTime)
```

Comment:
In CalcFlightTime() the speed must be divided by 10 `float64(speed/10)` this should fix the calculation finally.